diff --git a/Dockerfile b/Dockerfile
index 144e856..219529b 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -32,6 +32,6 @@ CMD echo ">>> Setting recalbox version to ${RECALBOX_VERSION}" && echo "${RECALB
     echo ">>> Reseting buildroot submodule" && ( cd buildroot && git reset HEAD --hard && git clean -dfx ) && \
     echo ">>> Fetching buildroot submodule" && ( git submodule update --init buildroot ) && \
     echo ">>> Making recalbox-${ARCH}_defconfig" && make recalbox-${ARCH}_defconfig && \
-    export RECALBOX_CCACHE=${RECALBOX_CCACHE_ENABLED:+"BR2_CCACHE=y BR2_CCACHE_DIR=/share/ccache BR2_CCACHE_INITIAL_SETUP=--max-size=500G BR2_CCACHE_USE_BASEDIR=y"} && \
+    export RECALBOX_CCACHE=${RECALBOX_CCACHE_ENABLED:+"BR2_CCACHE=y BR2_CCACHE_DIR=/share/ccache BR2_CCACHE_INITIAL_SETUP=--max-size=${RECALBOX_CCACHE_MAX_SIZE:-500G} BR2_CCACHE_USE_BASEDIR=y"} && \
     echo ">>> Make with command : BR2_DL_DIR="/share/dl" $RECALBOX_CCACHE $PACKAGE" && \
     make BR2_DL_DIR="/share/dl" $RECALBOX_CCACHE $PACKAGE
