diff --git a/package/libretro-tamalibretro/libretro-tamalibretro.mk b/package/libretro-tamalibretro/libretro-tamalibretro.mk
index b4c3c00..ef06a63 100644
--- a/package/libretro-tamalibretro/libretro-tamalibretro.mk
+++ b/package/libretro-tamalibretro/libretro-tamalibretro.mk
@@ -30,7 +30,8 @@ define LIBRETRO_TAMALIBRETRO_BUILD_CMDS
 	CFLAGS="$(TARGET_CFLAGS) $(COMPILER_COMMONS_CFLAGS_SO)" \
 		CXXFLAGS="$(TARGET_CXXFLAGS) $(COMPILER_COMMONS_CXXFLAGS_SO)" \
 		LDFLAGS="$(TARGET_LDFLAGS) $(COMPILER_COMMONS_LDFLAGS_SO)" \
-		$(MAKE) CXX="$(TARGET_CXX)" CC="$(TARGET_CC)" -C $(@D)/ -f Makefile platform="unix"
+		$(MAKE) CXX="$(TARGET_CXX)" CC="$(TARGET_CC)" -C $(@D)/ -f Makefile platform="unix" \
+		GIT_VERSION="$(shell echo $(LIBRETRO_TAMALIBRETRO_VERSION) | cut -c 1-8)"
 endef
 
 define LIBRETRO_TAMALIBRETRO_INSTALL_TARGET_CMDS
