diff --git a/package/libretro-melonds/libretro-melonds.mk b/package/libretro-melonds/libretro-melonds.mk
index fe3d002..e54f4ff 100644
--- a/package/libretro-melonds/libretro-melonds.mk
+++ b/package/libretro-melonds/libretro-melonds.mk
@@ -22,7 +22,7 @@ define LIBRETRO_MELONDS_BUILD_CMDS
 		CXXFLAGS="$(TARGET_CXXFLAGS) $(COMPILER_COMMONS_CXXFLAGS_SO) $(LIBRETRO_MELONDS_PIC)" \
 		LDFLAGS="$(TARGET_LDFLAGS) $(COMPILER_COMMONS_LDFLAGS_SO) $(LIBRETRO_MELONDS_PIC)" \
 		$(MAKE) CXX="$(TARGET_CXX)" CC="$(TARGET_CC)" -C $(@D)/ -f Makefile platform="$(LIBRETRO_MELONDS_PLATFORM)" \
-		GIT_VERSION="$(shell echo $(LIBRETRO_PMELONDS_VERSION) | cut -c 1-8)"
+		GIT_VERSION="$(shell echo $(LIBRETRO_MELONDS_VERSION) | cut -c 1-8)"
 endef
 
 define LIBRETRO_MELONDS_INSTALL_TARGET_CMDS
