diff --git a/package/megatools/megatools.mk b/package/megatools/megatools.mk
index 6e4fc8f..9f2b055 100644
--- a/package/megatools/megatools.mk
+++ b/package/megatools/megatools.mk
@@ -5,7 +5,8 @@
 ################################################################################
 
 MEGATOOLS_VERSION = 1.10.3
-MEGATOOLS_SITE = https://megatools.megous.com/builds
+MEGATOOLS_SITE = http://deb.debian.org/debian/pool/main/m/megatools
+MEGATOOLS_SOURCE = megatools_$(MEGATOOLS_VERSION).orig.tar.gz
 MEGATOOLS_LICENSE = GPLv2
 
 MEGATOOLS_CONF_OPTS = \
