diff --git a/package/recalbox-themes/recalbox-themes.mk b/package/recalbox-themes/recalbox-themes.mk
index af59e83952..e14aba474c 100644
--- a/package/recalbox-themes/recalbox-themes.mk
+++ b/package/recalbox-themes/recalbox-themes.mk
@@ -8,7 +8,7 @@ ifneq ($(GITLAB_TOKEN_THEMES),)
 RECALBOX_THEMES_VERSION = febce03c483cca4fdcd873938e974e0a52930418
 RECALBOX_THEMES_SITE = https://gitlab-ci-token:$(GITLAB_TOKEN_THEMES)@gitlab.com/recalbox/recalbox-themes-prime
 else
-RECALBOX_THEMES_VERSION = 284af2f1a598ba5ff3c9fb32ab0509f6bbd6291f
+RECALBOX_THEMES_VERSION = 3ec0bcd63a159568845136f68c10008ec4362897
 RECALBOX_THEMES_SITE = https://gitlab.com/recalbox/recalbox-themes
 endif
 RECALBOX_THEMES_SITE_METHOD = git
@@ -18,8 +18,6 @@ define RECALBOX_THEMES_COMMON
 	mkdir -p $(TARGET_DIR)/recalbox/share_init/system/.emulationstation/themes/
 	cp -r $(@D)/themes/recalbox-next \
 		$(TARGET_DIR)/recalbox/share_init/system/.emulationstation/themes/
-	cp -r $(@D)/themes/recalbox-next-v9 \
-		$(TARGET_DIR)/recalbox/share_init/system/.emulationstation/themes/
 	cp -r $(@D)/themes/recalbox-goa2 \
 		$(TARGET_DIR)/recalbox/share_init/system/.emulationstation/themes/
 	cp -r $(@D)/themes/recalbox-240p \
